pkg/reconcilermanager/controllers/rootsync_controller.go

```diff
@@ -158,6 +159,14 @@ func (r *RootSyncReconciler) Reconcile(ctx context.Context, req controllerruntim
 		r.logger(ctx).V(3).Info("Sync deletion timestamp detected")
 	}
 
+	enabled, err := r.isWebhookEnabled(ctx)
+	if err != nil {
+		return controllerruntime.Result{}, err
+	}
+	if r.webhookEnabled != enabled {
+		r.webhookEnabled = enabled
```

Comment:
Inconsistency between rootsync_controller.go and reposync_controller.go.
It's okay to assign without the equality check.